README.md

```diff
@@ -41,21 +42,29 @@ Follow these steps to open this sample in a container using the VS Code Dev Cont
 
 Once you have this sample opened, you'll be able to work with it like you would locally.
 
-> **Note:** This container runs as a non-root user with sudo access by default. Comment out `"remoteUser": "vscode"` in `.devcontainer/devcontainer.json` if you'd prefer to run as root.
-
 Some things to try:
 
 1. **Edit:**
    - Open `main.cpp`
    - Try adding some code and check out the language features.
-   - Notice that the C++ extension is already installed in the container since the `.devcontainer/devcontainer.json` lists `"ms-vscode.cpptools"` as an extension to install automatically when the container is created.
+   - Make a spelling mistake and notice it is detected. The [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) extension was automatically installed because it is referenced in `.devcontainer/devcontainer.json`.
+   - Also notice that utilities like `Vcpkg` and the [C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) extension are installed. Tools are installed in the `mcr.microsoft.com/devcontainers/cpp` image and Dev Container settings and metadata are automatically picked up from [image labels](https://containers.dev/implementors/reference/#labels).

```

Comment:
See previous comments on what software should be installed. The current head of the branch removes the C++ extension.

Reply:
To recap:
1. The second note is highlighting the new image metadata capabilities that brings these extensions across without being in devcontainer.json.  See https://containers.dev/implementors/reference/#labels
2. Happy to have another extension, but this spell checker is present to illustrate that you can supplement image metadata with contents from devcontainer.json.